eogrow/tasks/features.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, validity_threshold: float):
 
     def __call__(self, array: np.ndarray) -> bool:
         coverage = np.sum(array.astype(np.uint8)) / np.prod(array.shape)
-        return bool(coverage > self.validity_threshold)  # numpy return np.bool_ by default
+        return (coverage > self.validity_threshold).astype(bool)
```

Comment:
I remember some fixes in `numpy` where `numpy` changed `np.bool` to `bool`. Could it be that the `numpy` version is some older one and this wouldn't be necessary with a newer version in the first place?

Reply:
god knows, we are stuck with the old version due to `numba`